lib/containers/urls.pm

```diff
@@ -342,4 +299,29 @@ sub get_3rd_party_images {
     return (\@images);
 }
 
+# Return url of specified image as a string.
+# Specify released => 0 for unreleased image.
+# Die when nothing to return.
+sub get_container_url {
```

Comment:
I would prefer to have a name that implies an `image` and not a `container`.
e.g. `get_image_url` or `get_container_image_url`